packages/rocketchat-slashcommands-leave/leave.js

```diff
@@ -0,0 +1,31 @@
+
+/*
+ * Leave is a named function that will replace /leave commands
+ * @param {Object} message - The message object
+ */
+	if (Meteor.isClient) {
+		RocketChat.slashCommands.add('leave', null, {
```

Comment:
because in this case, there are no difference.
the test is ```RocketChat.slashCommands.commands[command]?.callback?``` it will test ```callback != null;```
, but I can change if you prefer.